src/plugins/data/common/search/tabify/tabify.ts

```diff
@@ -146,13 +146,28 @@ export function tabifyAggResponse(
   }
 
   const write = new TabbedAggResponseWriter(aggConfigs, respOpts || {});
+  // This is a very special case where
+  const hasMultipleDocCountAtRootWithFilters = Object.keys(esResponse.aggregations ?? {}).some(
+    (key) => /doc_count_/.test(key)
+  );
+
+  const aggsRoot = aggConfigs.isSamplingEnabled()
+    ? esResponse.aggregations?.sampling
+    : esResponse.aggregations;
   const topLevelBucket: AggResponseBucket = {
-    ...(aggConfigs.isSamplingEnabled()
-      ? esResponse.aggregations?.sampling
-      : esResponse.aggregations),
-    doc_count: esResponse.aggregations?.doc_count || esResponse.hits?.total,
+    ...aggsRoot,
+    doc_count: aggsRoot?.doc_count,
   };
 
+  // The fix itself is one line, but it's a bit hard to clear assess the full impact of it
+  // therefore here's a check to scope down the impact to the known scenario with the bug
+  // this can be lifted off once the full impact is assessed
```

Comment:
This comment seems more like a PR review comment... if it's intended to stay can it link to the issue or describe it in more detail? 😊